extensions/data-workspace/src/main.ts

```diff
@@ -42,10 +42,6 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 	const dataWorkspaceExtension = new DataWorkspaceExtension(workspaceService);
 	Logger.log(`DataWorkspaceExtension constructor took ${new Date().getTime() - dataWorkspaceExtensionStartTime}ms`);
 
-	const registerTreeDataProvidertartTime = new Date().getTime();
-	context.subscriptions.push(vscode.window.registerTreeDataProvider('dataworkspace.views.main', workspaceTreeDataProvider));
```

Comment:
I removed this because `vscode.window.createTreeView()` needs to be used to be able to specify the `dragAndDropController`, which is now being done in the constructor of `WorkspaceTreeDataProvider`.